Add make targets for deploying

aleksanb opened the pull request.

aleksanb committed:
Add make targets for deploying

sigvef commented on Makefile:
```diff
@@ -20,3 +20,11 @@ copy-travis-files:
 test:
 	bundle exec rubocop -R -D
+
+.PHONY:
```
Makefile phony syntax looks like this: `.PHONY: deploy-production` :)

aleksanb replied:
Huh! So has make been creating secret shadow files all along or something?
The command still works even though phony is incorrect.

sigvef replied:
No, make won't make files based on target names automatically, but if someone were to manually create a file with the same name as a target (e.g. `test`) then make will stop making that target.

aleksanb committed:
Fix incomplete .PHONY syntax

.PHONY now actually works.
Make should no longer look for files with corresponding names to the make tasks.

aleksanb commented:
Ping.
